element-template-generator/postman-collections-parser/src/test/java/io/camunda/connector/generator/postman/PostmanCollectionsGenerationSourceTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.generator.postman;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.camunda.connector.generator.api.GeneratorConfiguration;
+import io.camunda.connector.generator.api.GeneratorConfiguration.ConnectorMode;
+import io.camunda.connector.generator.postman.utils.ObjectMapperProvider;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class PostmanCollectionsGenerationSourceTest {
+
+  @Test
+  void testNew() throws JsonProcessingException {
+    var source = new PostmanCollectionsGenerationSource(List.of("src/main/resources/docusign.json", "diagnostics/Gets the API request logging log files.", "accounts/creates new ACCOUNTS.", "Authentication/01 Authorize Code Grant Access Token"));
+    var gen = new PostmanCollectionOutboundTemplateGenerator();
+    var templates = gen.generate(source, new GeneratorConfiguration(ConnectorMode.NORMAL, null, null, 1, null, List.of()));
+    var resultString =
+        ObjectMapperProvider.getInstance()
+            .writerWithDefaultPrettyPrinter()
+            .writeValueAsString(templates);
+    System.out.println(resultString);
+    Assertions.assertThat(source).isNotNull();
```

Comment:
Would be great to add some assertions like properties that are expected as well as operations. WDYT?

Reply:
@sbuettner, yes, sure, this is early draft, tests will be covered in much more extensive way once I resolve all basic outstanding challenges: CLI, authentication, variable handling, body, etc.
For visibility: @bastiankoerber is taking a look at early generator so that I can get an early feedback.

Reply:
Great, thanks for the context @igpetrov

Reply:
Done and more to come.